lib/fedora_migrate/migration_options.rb

```diff
@@ -8,6 +8,7 @@ def conversion_options
     end
 
     def forced?
+      return false if options.nil?
```

Comment:
You could perhaps streamline and DRY `forced?` and `application_creates_versions?` thusly:

```ruby
def forced?
  option_set?(:force)
end

def application_creates_versions?
  option_set?(:application_creates_versions)
end

private

def option_set?(name)
  !!(options && options[name]) || false
end
```